toketmaster/tests/test_signing.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+
+import pytest
+from nacl.exceptions import BadSignatureError
+
+from toketmaster.src import models, message
+
+
+class TestSignature(unittest.TestCase):
+    def setUp(self) -> None:
+        self.client_wallet = models.Wallet(
+            public_key='CVFHOVYERCILMXWUBJZKMINGFF5OJ5A5QDMVXSDZL2L5DEIVZCKLP2XIBI',
```

Comment:
[Accounts fixture](https://github.com/algocertified/sandbox/blob/master/toketmaster/fixtures.py#L21) is available for this.

Reply:
Are these on the sandbox net or the test net? Why is NetworkAccounts doing so much stuff when we only need 2 accounts pre-fauceted with Algos?